build-notest.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+npm run bootstrap && npm run doc && npm run depcheck && npm run prettier && npm run lint && npm run rollup && ./publish-es6.sh && ./publish-wasm.sh && echo 'DONE'
```

Comment:
Is this an overall build script?

Reply:
build.sh rebuilds to generate a PR that should pass.
build-notest.sh does the same, but skips the test step -- the regression notebooks are run.